### PR TITLE
Ackwait configuration for Jetstream

### PR DESCRIPTION
**Pull request by Joibel**
Add the ability to configure the ackwait timeout for the consumer.

We can take a while processing messages, so need this configurable.

### CODE REVIEW

File: nats-jetstream-http-connector/README.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ The job of the connector is to read messages from the subject in the given strea
...
 - `ACCOUNT`: Name of the NATS account. `$G` is default when no account is configured.
+- `ACKWAIT`: A time.Duration formatted string for how long to wait for an acknowledgement that a message has been processed. Defaults to `30s`. Cannot be modified on a durable consumer without manually deleting the consumer.
 
 ## Resources
 
 - To setup and run nats streaming server, reference <https://docs.nats.io/nats-server/installation#installing-on-kubernetes-with-nats-operator>
-- For running the connecter with fission e.g.  
+- For running the connecter with fission e.g.
 
 ```fission mqt create --name jetstreamtest --function helloworld --mqtype nats-jetstream --mqtkind keda --topic input.created --resptopic output.response-topic --errortopic erroutput.error-topic --maxretries 3 --metadata stream=input --metadata natsServerMonitoringEndpoint=nats-jetstream.default.svc.cluster.local:8222 --metadata natsServer=nats://nats-jetstream.default.svc.cluster.local:4222 --metadata consumer=fission_consumer --metadata account=\$G```
```
Comment (neha-Gupta1) [inline nats-jetstream-http-connector/README.md:24]: Please add `ACKWAIT` usage in the command example.

Comment (Joibel) [reply nats-jetstream-http-connector/README.md]: Done.

**Comment by Joibel** [comment]
PR for fission created here https://github.com/fission/fission/pull/2575.

I hope that's everything needed.

**Review by sanketsudake — APPROVED** [review]

**Comment by sanketsudake** [comment]
Thanks for contribution @Joibel 🎉

**Pull request merged**